src/autoComplete/index.unit.tsx

```diff
@@ -47,34 +46,33 @@ describe('AutoComplete', () => {
 
   it('Renders with a custom className', () => {
     const customClassName = 'my-search'
-    const wrapper = shallow(<AutoComplete
-      {...defaultProps}
-      className={customClassName}
-    />)
+    const wrapper = shallow(<AutoComplete {...defaultProps} className={customClassName} />)
     expect(wrapper.hasClass(customClassName)).toBe(true)
   })
 
   it('Renders a title on the close button', () => {
-    const wrapper = mount(<AutoComplete
-      {...defaultProps}
-      buttonTitle="buttonTitle"
-    />)
+    const wrapper = mount(<AutoComplete {...defaultProps} buttonTitle="buttonTitle" />)
     expect(wrapper.find('button').prop('title')).toBe('buttonTitle')
   })
 
   it('Can trigger onDoneAnimationEnd callback', () => {
     const event = jest.fn()
-    const wrapper = mount(<AutoComplete
-      {...defaultProps}
-      onDoneAnimationEnd={event}
-      selectedItemStatus={ItemChoiceStatus.CHECKED}
-    />)
+    const wrapper = mount(
+      <AutoComplete
+        {...defaultProps}
+        onDoneAnimationEnd={event}
+        selectedItemStatus={ItemChoiceStatus.CHECKED}
+      />,
+    )
     wrapper.instance().onInputChange({ value: 'title' })
 
     const items = fakeSearchForItems()
     wrapper.setProps({ isSearching: true })
```

Comment:
Why do you simulate the loading state in the proprs change since it's not the feature you are testing

Reply:
I didn't touch this code...

Reply:
Yep sorry, just saw that. I put a global comment regarding these tests afterwards